.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "secp256k1Native/secp256k1"]
```

Comment:
Oh.  Has there been a discussion about using submodules?  

I don't mind them, but I generally only use them in team settings where everybody is OK with them and are reasonably comfortable using them.

Reply:
I thought that was the most reasonable thing to do in this case.